### PR TITLE
Change to OCS 4.4 as default

### PR DESCRIPTION
**Pull request by petr-balogh**
Infra ticket:
https://projects.engineering.redhat.com/browse/OCSQE-320

Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

File: ocs_ci/framework/conf/default_config.yaml [hunk 1]
```diff
@@ -98,12 +98,12 @@ ENV_DATA:
...
   skip_ocp_deployment: false
   skip_ocs_deployment: false
-  ocs_version: '4.3'
+  ocs_version: '4.4'
   # uncomment to use custom directory for storing measurement data related to
   # monitoring tests, otherwise generate temporary directory for each test run
   # measurement_dir: '/tmp/ocs_ci_monitoring_measurement/'
   # Default RHCOS image to be used for VmWare deployment
-  vm_template: 'rhcos-4.3.0-x86_64-vmware'
+  vm_template: 'rhcos-4.4.0-x86_64-vmware'
```
Comment (vavuthu) [inline ocs_ci/framework/conf/default_config.yaml:106]: didn't see above template in in downloads page. I see below as latest
rhcos-4.4.0-rc.1-x86_64-vmware.x86_64

Comment (petr-balogh) [reply ocs_ci/framework/conf/default_config.yaml]: done

**Comment by petr-balogh** [comment]
Running verification on DC 26:  https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/6842/console

**Comment by petr-balogh** [comment]
Attempt above failed on issue:
```
time="2020-04-21T11:57:12Z" level=info msg="Waiting up to 30m0s for the cluster at https://api.pbalogh-44.qe.rh-ocs.com:6443 to initialize..."
time="2020-04-21T11:57:16Z" level=debug msg="Still waiting for the cluster to initialize: Working towards 4.4.0-0.nightly-2020-04-21-033933: 97% complete"
time="2020-04-21T11:57:55Z" level=debug msg="Still waiting for the cluster to initialize: Working towards 4.4.0-0.nightly-2020-04-21-033933: 99% complete"
time="2020-04-21T11:58:55Z" level=debug msg="Still waiting for the cluster to initialize: Working towards 4.4.0-0.nightly-2020-04-21-033933: 99% complete, waiting on authentication, console, csi-snapshot-controller, ingress, kube-storage-version-migrator, monitoring"
time="2020-04-21T12:02:10Z" level=debug msg="Still waiting for the cluster to initialize: Some cluster operators are still updating: authentication, console, csi-snapshot-controller, ingress, kube-storage-version-migrator, monitoring"
time="2020-04-21T12:04:03Z" level=debug msg="Still waiting for the cluster to initialize: Some cluster operators are still updating: authentication, console, csi-snapshot-controller, ingress, kube-storage-version-migrator, monitoring"
```
Full log from installer: http://magna002.ceph.redhat.com/ocsci-jenkins/openshift-clusters/pbalogh-44/pbalogh-44_20200421T101333/logs/openshift_install_create_cluster_1587472385.log


Found similar issue here but here it's related to azure:
https://github.com/openshift/installer/issues/2334

Trying again but build is now blocked by PSI issue.
Should be triggered as https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/6​85​2 but now waiting in queue.

As I see it collected must gather I will try to open BZ for the issue we hit.

**Comment by petr-balogh** [comment]
Opened BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1826390

**Comment by prsurve** [comment]
@petr with the same build I was able to create ocp as well as ocs on BM

**Comment by petr-balogh** [comment]
> I had seen this when worker node were not created or the might have created late so csr approval is required can you check if the worker node created

Thanks, the cluster got removed right after. I re-triggered the job if I will see this again.

**Comment by petr-balogh** [comment]
Now verification is blocked cause of change in installer repo and issue:  https://github.com/red-hat-storage/ocs-ci/issues/1884

**Comment by petr-balogh** [comment]
Next one verification:
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/6886/console

Previous failed to download the OCP client :(

**Comment by petr-balogh** [comment]
The verification is failing with the issue: https://projects.engineering.redhat.com/browse/OCSQE-321 . So to get this merged I moved the default OCP to 4.3 for now till it's fixed. So consider as verified.

**Pull request merged**